packages/app-accounts/src/Backup.tsx

```diff
@@ -37,11 +39,14 @@ class Backup extends React.PureComponent<Props, State> {
   }
 
   render () {
+    const { className } = this.props;
+
     return (
       <Modal
-        size='tiny'
+        className={classes('accounts--Backup', className)}
```

Comment:
Actually specifically left this `classes(...)` dance out. We are using it everywhere in app-* and it actually serves no benefit since all components are used and instantiated once, with no sharing. So really removed most/all where I could lay my hands on (in app-*). In cases where it is needed, can be added.

Obviously, for ui-*, it is very much needed since they are all _shared_ components.